CHANGELOG.next.asciidoc

```diff
@@ -66,6 +66,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Metricbeat*
 
 - Add _bucket to histogram metrics in Prometheus Collector {pull}11578[11578]
+- Stop overwriting cloud.* fields from add_cloud_metadata if they are not empty. {pull}11612[11612] {issue}11305[11305]
```

Comment:
If not sure mine is any more clear, so use your judgement. 😄 

```suggestion
- Change `add_cloud_metadata` processor to not overwrite the `cloud` field when it is already present in the event. {pull}11612[11612] {issue}11305[11305]
```

Reply:
Yours is a lot clearer 😄 Thanks!